packages/apps/admin/changelog.md

```diff
@@ -1,3 +1,7 @@
+1.0.124
+- show time in 24h format for all locales
+- show time in 24h format for all locales
```

Comment:
todo: this change is not good

somehow merging or rebasing went wrong.. these changes should not be part of your PR